components/DocTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Link from 'next/link';
+
+const DocTable = ({ frontmatter }: any) => {
+  return (
+    <>
+      <div className='max-w-full mx-auto overflow-auto'>
+        <table className='table-auto border-collapse w-full bg-slate-200 dark:bg-slate-900 text-slate-700 dark:text-slate-300'>
+          <tbody>
+            <tr className='dark:hover:bg-slate-950 hover:bg-slate-300'>
+              <td className='border border-slate-400 dark:border-slate-500 p-2 text-center'>
+                Specification
```

Comment:
```suggestion
                **Specification**
```

Let's make bold each property name.